syntax = "proto3";

package helloworld;

service Greeter {
  rpc SayHello (HelloRequest) returns (HelloReply) {}
}

message HelloRequest {
  int32 connection = 1;
}

message HelloReply {
  int32 connection = 1;
}
